src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -1780,7 +1783,7 @@ public class LibraryServiceApi implements AutoCloseable {
    *   BookName namesElement = BookName.create("[SHELF_ID]", "[BOOK_ID]");
    *   List&lt;BookName&gt; names = Arrays.asList(namesElement);
    *   List&lt;ShelfName&gt; shelves = new ArrayList&lt;&gt;();
-   *   for (BookName element : libraryServiceApi.findRelatedBooks(names, shelves).iterateAllElements()) {
+   *   for (String element : libraryServiceApi.findRelatedBooks(names, shelves).iterateAllElements()) {
```

Comment:
I think the example loop should prefer the typed resource name instead of string